Feat/new-links-1858: added additional links to external resources (#1897)
grommet/hpe-design-system: aries-site/src/pages/foundation/accessibility.mdx

For guidance on setting up your development environment, see [Developer Guidance](/foundation/developer-guidance).

## Internal Resources

- [HPE Video Guides on Accessibility (HPE internal link)](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-design-training.aspx): A collection of educational videos from HPE Product Accessibility Program Office to assist all stakeholders in understanding and implementing accessible design.
- [HPE Accessible Web, Software & Hardware Design Guides (HPE internal link)](https://hpe.sharepoint.com/sites/f5/cto/office/Accessibility/Pages/accessible-web-software-and-hardware-design.aspx): Text-based guides from HPE Product Accessibility Program Office on designing and implementing accessible products.

## External Resources

- [WCAG 2.1](https://www.w3.org/TR/WCAG21/): A robust set of accessibility design guidelines for software and web-based products. HPE policy is to conform to all applicable guidelines identified as Level A and AA.

> - [WebAIM Introduction to Web Accessibility](https://webaim.org/intro/): An all-encompassing introducation to how accessibility can improve the web for all users.

Reviewer: ```suggestion
- [WebAIM Introduction to Web Accessibility](https://webaim.org/intro/): An introduction on how accessibility improves the Web for all users.
```

> - [The A11y Project Checklist](https://www.a11yproject.com/checklist/): A checklist that uses WCAG 2.1 as a reference for developers and designers to ensure their product's accessibility. 

Reviewer: ```suggestion
- [The A11y Project Checklist](https://www.a11yproject.com/checklist/): A checklist using WCAG 2.1 as a reference for developers and designers to ensure their product's accessibility. 
```
